document_merge_service/api/unoconv.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+from mimetypes import guess_type
+
+
+class Unoconv:
+    def __init__(self, convert, filepath):
+        self.convert = convert
+        self.filepath = filepath
+        self.content_type, _ = guess_type(f"something.{convert}")
+        self.result = None
+        self.returncode = None
+
+    def process(self):
+        p = subprocess.run(
+            [
+                "/usr/bin/python3.5",
```

Comment:
Does this hard-link the implementation to python 3.5?
(I'm not familiar with the subprocess API)

Reply:
It took me way too long to get this running. Turns out, that unoconv MUST be running with the same python version as libreoffice uses. This is in case of debian 3.5 (even in a 3.6 container).

Reply:
Let's add some documentation about this then :wink: (Comment or even README)

Reply:
I would consider putting the path of the required python interpreter and a check that verifies if the path exists to the settings.py to avoid runtime errors.

Reply:
Done!